src/views/v2/month/event-multiday-spacer.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * View: Month Event Multiday - Hidden Event / Spacer
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/event-multiday.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+
+?>
+<div class="tribe-events-calendar-month__multiday-wrapper">
```

Comment:
we don't need 2 divs here, we just need 1 with the right padding/height and margin